app/queries/get_host_content.rb

```diff
@@ -109,9 +112,9 @@ def clauses
       clauses = TABLES[:editions][:state].eq(state)
                                          .and(TABLES[:links][:link_type].eq(embedded_link_type))
                                          .and(TABLES[:links][:target_content_id].eq(target_content_id))
-      if host_content_id
-        clauses = clauses.and(TABLES[:documents][:content_id]).eq(host_content_id)
-      end
+
+      clauses = clauses.and(TABLES[:documents][:content_id]).eq(host_content_id) if host_content_id
```

Comment:
Can you explain why this `host_content_id` has moved? Wondering if that clauses assignment not longer being covered will cause issues

Reply:
Oh, I just changed it to an inline if to make the assignment a bit cleaner

Reply:
Oh yeah my bad, github review format tricking me, it's not the first time 🤦